llvm/unittests/TargetParser/TargetParserTest.cpp

```diff
@@ -1841,7 +1868,8 @@ INSTANTIATE_TEST_SUITE_P(
                  AArch64::AEK_PROFILE, AArch64::AEK_RAND,
                  AArch64::AEK_FP16FML, AArch64::AEK_I8MM,
                  AArch64::AEK_JSCVT,   AArch64::AEK_FCMA,
-                 AArch64::AEK_PAUTH,   AArch64::AEK_PERFMON}),
+                 AArch64::AEK_PAUTH,   AArch64::AEK_PERFMON,
+                 AArch64::AEK_CCDP}),
```

Comment:
We should think about whether we want each AEK to correspond to a `-march` modifier (status quo) or whether we want it to correspond to an `Extension` like here.

Reply:
I don't see an advantage to not having AEK entries for all extensions. Having this correspondence not only increases testability, as shown by the changes in `TargetParserTest.cpp`, but also makes them consistently visible to the wider codebase via the Target Parser interface.